frontend/packages/console-shared/src/components/status/status.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import * as History from 'history';
+import { Link } from 'react-router-dom';
+import { Popover } from '@patternfly/react-core';
+import { Icon, Button } from 'patternfly-react';
+
+type StatusProps = {
+  icon?: string;
+  children: React.ReactNode;
+};
+
+export const Status = ({ icon, children }: StatusProps) => (
+  <>
+    {icon && <Icon type="pf" name={icon} className="kubevirt-status__icon" />} {children}
+  </>
+);
+
+type PopoverStatusProps = StatusProps & {
+  header: React.ReactNode;
+};
+
+export const PopoverStatus = ({ icon, header, children }: PopoverStatusProps) => (
+  <Popover position="right" headerContent={header} bodyContent={children}>
+    <span className="kubevirt-status__popover">
+      <Status icon={icon}>
+        <Button className="kubevirt-status__button" bsStyle="link">
```

Comment:
A button without any event handling?

Reply:
The Button is internally enhanced by Popover and acts as a toggle (https://patternfly-react.surge.sh/patternfly-4/components/popover/)

Reply:
oh pf...